__playwright-tests__/forms.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { test, expect } from '../src';
 
 test('form submits succesfully', async ({ page }) => {
-  test.setTimeout(2000);
+  test.setTimeout(5000);
```

Comment:
Is this bump in the timeout necessary? I think we should be able to remove this altogether now. I added it when this bug was still active because the form submission went into a redirect loop that would continue until the global test timeout was hit, so this was just to shorten the time to failure.

Reply:
I can try without it. I was getting timeout errors before I added that, but I don't know if I tried with the most recent branch, so if you fixed something we might be able to eliminate this.

Reply:
It works without the `setTimeout`, so I removed it.